.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+### Description
+
+### What works
+
+### What was not tested
+
+### Test
+Operating system (and linux kernel version): 
+Hardware (GPU/CPU):
+
+### Ready for review
+[ ] Script tested as a regular phoenicis user and working (if you have a problem -> draft and ask for help).
```

Comment:
I think this is more readable with bullets (-/*), i.e.:

- [ ] ...
- [ ] ...
- [ ] ...

Reply:
This problem is still there. To see what I mean take a look how the document is rendered.